### PR TITLE
Mention section on posting phase

### PR DESCRIPTION
**Pull request by mcass19**

### CODE REVIEW

File: app/assets/stylesheets/retrospective.scss [hunk 1]
```diff
@@ -11,25 +11,35 @@
...
 }
 
-.discuss_section__title {
+.posting_phase__section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.posting-phase__title {
```
Comment (fchavat) [inline app/assets/stylesheets/retrospective.scss]: `posting-phase_title` is not following the same naming convention used in the others selectors (`posting_phase_x`)

Comment (mcass19) [reply app/assets/stylesheets/retrospective.scss]: Ooooh I forgot change that 😞. 

Fixed in 32bdbf6 😃.

File: app/assets/stylesheets/retrospective.scss [hunk 2]
```diff
@@ -11,17 +11,27 @@
...
+  &__section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
 
   &__title {
     font-size: 20px;
+
+    &.mention {
```
Comment (fchavat) [inline app/assets/stylesheets/retrospective.scss]: I think that `&` is not needed here.
What I think you are trying to generate is `.posting_phase__title .mention`, and what you are actually generating is `.posting_phase__title.mention`
[Here](https://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_rules) you can find some good information about it 👍

Comment (mcass19) [reply app/assets/stylesheets/retrospective.scss]: We already discussed it 💪!

**Pull request merged**